samples/client/petstore/php/SwaggerClient-php/lib/Api/FakeApi.php

```diff
@@ -1075,6 +1075,237 @@ protected function fakeOuterStringSerializeRequest($body = null)
         );
     }
 
+    /**
+     * Operation testBodyWithQueryParams
+     *
+     * @param  \Swagger\Client\Model\User $body body (required)
+     * @param  string $query query (required)
+     *
+     * @throws \Swagger\Client\ApiException on non-2xx response
+     * @throws \InvalidArgumentException
+     * @return void
+     */
+    public function testBodyWithQueryParams($body, $query)
+    {
+        $this->testBodyWithQueryParamsWithHttpInfo($body, $query);
+    }
+
+    /**
+     * Operation testBodyWithQueryParamsWithHttpInfo
+     *
+     * @param  \Swagger\Client\Model\User $body (required)
+     * @param  string $query (required)
+     *
+     * @throws \Swagger\Client\ApiException on non-2xx response
+     * @throws \InvalidArgumentException
+     * @return array of null, HTTP status code, HTTP response headers (array of strings)
+     */
+    public function testBodyWithQueryParamsWithHttpInfo($body, $query)
+    {
+        $returnType = '';
+        $request = $this->testBodyWithQueryParamsRequest($body, $query);
+
+        try {
+            $options = $this->createHttpClientOption();
+            try {
+                $response = $this->client->send($request, $options);
+            } catch (RequestException $e) {
+                throw new ApiException(
+                    "[{$e->getCode()}] {$e->getMessage()}",
+                    $e->getCode(),
+                    $e->getResponse() ? $e->getResponse()->getHeaders() : null,
+                    $e->getResponse() ? $e->getResponse()->getBody()->getContents() : null
+                );
+            }
+
+            $statusCode = $response->getStatusCode();
+
+            if ($statusCode < 200 || $statusCode > 299) {
+                throw new ApiException(
+                    sprintf(
+                        '[%d] Error connecting to the API (%s)',
+                        $statusCode,
+                        $request->getUri()
+                    ),
+                    $statusCode,
+                    $response->getHeaders(),
+                    $response->getBody()
+                );
+            }
+
+            return [null, $statusCode, $response->getHeaders()];
+
+        } catch (ApiException $e) {
+            switch ($e->getCode()) {
+            }
+            throw $e;
+        }
+    }
+
+    /**
+     * Operation testBodyWithQueryParamsAsync
+     *
+     * 
+     *
```

Comment:
There's extra whitespace lines here. Can they be removed?

Reply:
Yeah, I'd imagine so but that file is actually auto generated by the bin scripts. The only real change in this PR is in [this commit](https://github.com/swagger-api/swagger-codegen/pull/8144/commits/215e6eaf5606c828096f9626ea19b6a21d2e0062). The other two commits are unit testing and a re-run of the sample generator (which produced this code).